### DISCUSSION TITLE
SEB LOCKED

### DISCUSSION THREAD

**Discussion by blueberries16** [SafeExamBrowser/seb-win-refactoring#856] [Q&A]
When I open SEB, the screen is fully red and it says SEB LOCKED on screen. “**The last session with the currently active configuration or start URL was not terminated properly! please enter the correct password to unlock SEB**”. I've tried reinstalling and it doesn't work. please give me a solution because I want to take an exam😭😭😭
![Uploading IMG_9385.jpeg…]()

**Comment by dbuechel** [comment]
You'd need to resolve this with your exam provider, they should know the correct unlock password.
